template/web/gatsby-config.js

```diff
@@ -7,18 +7,20 @@ const clientConfig = require('./client-config')
 const token = process.env.SANITY_READ_TOKEN
 
 const isProd = process.env.NODE_ENV === 'production'
+const previewEnabled = (process.env.GATSBY_IS_PREVIEW || "false").toLowerCase() === "true"
 
 module.exports = {
   plugins: [
     'gatsby-plugin-postcss',
     'gatsby-plugin-react-helmet',
+    'gatsby-plugin-image',
     {
       resolve: 'gatsby-source-sanity',
       options: {
         ...clientConfig.sanity,
         token,
-        watchMode: !isProd,
-        overlayDrafts: !isProd && token
+        watchMode: !previewEnabled,
```

Comment:
@kathmbeck with this value for `watchMode`, it'll be used in builds, which could lead to some issues... Could you change this to `!isProd && !previewEnabled`? Thanks 🙇‍♂️